spec/datadog/tracing/contrib/dalli/instrumentation_spec.rb

```diff
@@ -78,13 +78,7 @@
   end
 
   describe 'when multiplexed configuration is provided' do
-    let(:service_name) { 'multiplex-service' }
-
-    before do
```

Comment:
Is removing this block correct? My understanding is it wouldn't be testing multiplexing then. The test becomes a false negative.

Reply:
Yes you're right. These tests will fail, but if we include the new env vars with new tests then it should work out.

Reply:
Not sure I understand how you mean by work out. The test isn't testing multiplexing if it doesn't apply multiplexing?

Reply:
Nevermind, reverted back to the way it was before so no worries